risc0/zkvm/src/prove/mod.rs

```diff
@@ -213,6 +217,9 @@ pub trait Prover {
 }
 
 fn provers() -> HashMap<String, Rc<dyn Prover>> {
+    if cfg!(feature = "disable-dev-mode") && std::env::var("DEV_MODE").is_ok() {
+        panic!("zkVM: dev mode is disabled. unset DEV_MODE environment variable to produce valid proofs")
```

Comment:
Suggesting same addition as with other comment.
```suggestion
        panic!(""zkVM: Inconsistent settings -- please resolve. The DEV_MODE environment variable is set but dev mode has been disabled by feature flag. To produce valid proofs, unset the DEV_MODE environment variable. To enable dev mode and accept INSECURE, invalid proofs, remove the `disable-dev-mode` feature flag.")
```